src/components/anotherlist.js

```diff
@@ -1,198 +1,78 @@
-                                <div className="details">
-                                     <p><b>Docanut(Ogbono)</b><br />Location: Idumota, Lagos State<br />Name: My Store<br />#2,500</p>
-                                </div>
-                          </Col>
-                          <Col md={3}>
-                              <div className="details">
-                                    <img src='/images/ripeBanana.jpg' alt="banana" style={{width: '150px', height: '100px'}} />
-                              </div>
-                                                                     &nbsp;&nbsp;&nbsp;
-                              <div className="details">
-                                  <p><b>Ripe Bananas</b><br />Location: Ore, Ondo State<br />Name: My Store<br />#1,250</p>
-                              </div>
-                          </Col>
-                   </Row>
-                         <div className="link-to-more">
-                               <Link to='/more-products/'>
-                                    More
-                               </Link>
-                         </div>
-              </Jumbotron>
-        </Container>
+        <div>
+           <Jumbotron className="show-products-available" style={{backgroundColor: 'ash'}}>
+             <Row>
+                <Col md={3}>
+                    <div>
+                      <div  style={{backgroundColor: 'white'}}>
+                            <InputGroup>
+                                  <InputGroupAddon addonType="prepend">
+                                       <Button onClick={this.search}>ok</Button>
+                                  </InputGroupAddon>
+                                  <Input placeholder="search products" />
+                                 
+                            </InputGroup>
+                      </div>
+                       <br /><br />
+                      <div style={{backgroundColor: 'white'}} className="little-details">
+                             <FindOnLocation />
+                      </div>
+                    </div>
+                </Col> 
+               <Col md={7} style={{backgroundColor: 'white'}}>
+                  <div style={{ marginTop: '25px'}}>
+                        <h6 style={{textAlign: 'center', marginTop: "-9%", marginBottom: "4px"}}>
+                          <Link to={ProductsAvailable}>Products Available</Link>
+                        </h6>
+                 </div>
+                       <br />
+                       <Row>
+                         <Row1 />
+                       </Row>
+                       <br />
+                       <Row>
+                         <Row2/>
+                       </Row>
+                       <br />
+                       <Row>
+                         <Row3 />
+                       </Row>
+                       <br />
+                       <Row>
+                         <Row4 />
+                       </Row>
```

Comment:
Your `Row1/2/3/4` should already be contained within a `Row` component no need repeat yourself here multiple times. Your `line 45 - 60` should look something like 
```
<Row1 />
<Row2 />
<Row3/>
<Row4 />
```

Reply:
also remove the `<br />` use css to do the spacing

Reply:
Done